FlameBot.js

```diff
@@ -110,6 +110,18 @@ class FlameBot {
         return;
       }
 
+      if (/cyber/i.test(message.text)) {
+          let sticker;
+          if (Math.random() < 0.5) {
+              // CYBER ATTACKS AHEAD
+              sticker = new Sticker('CAADAgADnAADGW8XB9XpWOhkV96rAg');
+          } else {
+              // CYBER
+              sticker = new Sticker('CAADAgADlgADGW8XB1uhplTaDTPOAg');
+          }
+          this.reply(sticker, message);
```

Comment:
Shouldn't we `return` here? Otherwise one gets a cyber sticker and maybe a random response.

Reply:
True, good catch! Fixed.